README.md

```diff
@@ -345,6 +346,48 @@ To capture data from EventSources running in the same process as EventFlow, the
 
 (*) Either providerName, or providerGuid must be specified. When both are specified, provider GUID takes precedence.
 
+#### Application Insights input
+
+*Nuget package:* [**Microsoft.Diagnostics.EventFlow.Inputs.EventFlowTelementryProcessor**](https://www.nuget.org/packages/Microsoft.Diagnostics.EventFlow.Inputs.EventFlowTelemetryProcessor/)
+
+Application Insights input is designed for the following scenario:
+1. You have an application that uses Application Insights for monitoring and diagnostics.
+2. You want to send a portion of your Application Insights telemetry to some destination other than Application Insights (e.g. Azure EventHub or Elasticsearch; the assumption is there is an EventFlow output for where the data needs to go).
+
+*Usage*
+1. Add the `EventFlowTelemetryProcessor` to your Application Insights configuration file (it goes into `TelemetryProcessors` element):
+   ```xml
+   <ApplicationInsights xmlns="http://schemas.microsoft.com/ApplicationInsights/2013/Settings" >
+     <!-- ... -->
+     <TelemetryProcessors>
+        <!-- ... -->
+        <Add Type="Microsoft.Diagnostics.EventFlow.ApplicationInsights.EventFlowTelemetryProcessor, Microsoft.Diagnostics.EventFlow.Inputs.EventFlowTelemetryProcessor" />
+        <!-- ... -->
```

Comment:
Added

*Note that the order of telemetry processors does matter.* In particular, if the `EventFlowTelemetryProcessor` is placed before the Application Insights sampling processor, EventFlow will capture all telemetry, but if the `EventFlowTelemetryProcessor` is placed after the sampling processor, it will only "see" telemetry that was sampled in. For more information on configuring Application Insights see [Application Insights configuration documentation](https://docs.microsoft.com/azure/application-insights/app-insights-configuration-with-applicationinsights-config).